.github/workflows/build.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build - JVM
-        run: mvn -B formatter:validate install --file pom.xml
+        run: mvn -B formatter:validate verify --file pom.xml
         
       - name: Build - Native
         run: mvn -B verify -Pnative --file integration-tests/pom.xml
```

Comment:
This one can remain as `verify`, but it depends on artifacts as mentioned from the previous step.

Reply:
Right, because JVM tests were executed in the second step anyway I merged them in a single verify call